.github/ISSUE_TEMPLATE/PROPOSAL_TEMPLATE.md

```diff
@@ -5,20 +5,36 @@ title: '[PROPOSAL]'
 labels: proposal
 assignees: ''
 ---
+## What/Why
 ### What are you proposing?
 _In a few sentences, describe the feature and its core capabilities._
 
-### How did you come up with this proposal?
-_Highlight any research, proposals, requests, issues, forum posts, anecdotes that signal this is the right thing to build. Highlight opportunities for additional research._
+### What users have sked for this feature?
+_Include links to GitHub Issues, Forums, Stack Overflow, Twitter, Etc_
+
+### What problems are you trying to solve?
+_Summarize the core use cases and user problems and needs you are trying to solve. Describe the most important user needs, pain points and jobs as expressed by the user asks above. Template: When \<a situation arises> , a \<type of user> wants to \<do something>, so they can \<expected outcome>. (Example: When **searching by postal code**, **a buyer** wants to **be required to enter a valid code** so they **don’t waste time searching for a clearly invalid postal code.**)_
+
+### What is the developer experience going to be?
+_Does this have a REST API? If so, please describe the API and any impact it may have to existing APIs. In a brief summary (not a spec), highlight what new REST APIs or changes to REST APIs are planned. as well as any other API, CLI or Configuration changes that are planned as part of this feature._
+
+#### Are there any security considerations? 
+Describe if the feature has any security considerations or impact. What is the security model of the new APIs? Features should be integrated into the OpenSearch security suite and so if they are not, we should highlight the reasons here.
+
+#### Are there any breaking changes to the API
+_If this feature will require breaking changes to any APIs, ouline what those are and why they are needed_
```

Comment:
Recommend adding some verbiage about a path to breaking changes (e.g., add new API and deprecate old one).

Reply:
Thanks! updated to add additional clarity here